tmdb/tmdb.go

```diff
@@ -641,3 +648,11 @@ func (credits *Credits) GetWriters() []string {
 	}
 	return writers
 }
+
+func GetImageQualities() (posterQuality, fanArtQuality, logoQuality, thumbnailQuality string) {
```

Comment:
If would be easier to return a struct, that describes all art types per selected quality. 

For example:

```go
type ImageQualityIdentifier string
type ImageQualityBundle struct {
  Poster ImageQualityIdentifier
  FanArt ImageQualityIdentifier
  ...
}
```

Reply:
this is a good suggestion. thank you!

but if i do `type ImageQualityIdentifier string` then i also need to do type casting to string in all statements like
`item.Art.Thumbnail = ImageURL(episode.StillPath, string(imageQualities.Thumbnail))`

i hope it is ok to just use standard `string` instead of custom string type.

Reply:
> but if i do type ImageQualityIdentifier string then i also need to do type casting to string in all statements like
item.Art.Thumbnail = ImageURL(episode.StillPath, string(imageQualities.Thumbnail))

2 options:
1) `ImageURL` should accept `ImageQualityIdentifier` (as it is the thing it expects really)
2) Use `string` instead of `ImageQualityIdentifier`.

Both are fine at this point. Would accept any.

Reply:
ok i will use ImageQualityIdentifier and also i found that now we have
https://pkg.go.dev/net/url@go1.19beta1#JoinPath
so i will use it in ImageURL

Reply:
done